src/index.html

```diff
@@ -11,7 +11,131 @@
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            name = "age"
+            value = "12"
+            required
+            min="1"
+            max="100"
+          >
+        </label>
+        <label class="form-field">
+          Full date of birth:
+          <input type="date" name = "date of birth">
+        </label>
+        <label class="form-field">
+          I accept the term of the agreement
+          <input type="checkbox" name = "agreement">
+        </label>
+      </fieldset>
+      <fieldset class="box">
+        <legend>Registration</legend>
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="eMail"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label class="form-field">
+          Password:
+          <input
+            type="password"
+            name="password"
+            required
+            minlength="6"
+            maxlength="32"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="box">
+        <legend>
+          An interesting fact about you?
+        </legend>
+        <div class="form-field">
+          Do you love cats?
+          <label required>
+            <input
+              type="radio"
+              name="cats"
+              value="Yes"
+            >
+            Yes
+          </label>
+          <label required>
+            <input
+              type="radio"
+              name="cats"
+              value="No"
+            >
+            No
+          </label>
+        </div>
+        <label class="form-field">
+          What is your favorite color?
+          <input type="color" name="color">
+        </label>
+        <label class="form-field">
+          What time do you go to bed?
+          <input type="date" name="time">
+        </label>
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select
+            name="brand of cars"
+            required
+            multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+            <option value=" " disabled></option>
+          </select>
+        </label>
+        <label class="form-field">
+          How do you rate our work?
+          <input type="range">
+        </label>
+      </fieldset>
+      <fieldset class="box">
+        <legend class="head">
+          Additional info:
+        </legend>
+        <label class="form-field">
+          Comments:
+          <textarea
+            name="comments"
+          >
+          </textarea>
+        </label>
+        <label class="form-field">
+          Would you recommend us?
+          <select
+            name="recommend" required>
+            <option value="Yes" selected>yes</option>
+            <option value="No">No</option>
```

Comment:
```suggestion
            <option value="Yes" selected>
              Yes
            </option>
            <option value="No">
              No
            </option>
```
I think it will look better